cmd/nsd/main.go

```diff
@@ -66,6 +70,27 @@ func newApp(logger log.Logger, db dbm.DB, traceStore io.Writer) abci.Application
 	return app.NewNameServiceApp(logger, db)
 }
 
+func dumpSchema(cdc *amino.Codec) *cobra.Command {
+	converter := typescriptify.New()
+	dumpSchemaCmd := &cobra.Command{
+		Use:   "schema",
+		Short: "dump schema from the amino codec",
+		Run: func(cmd *cobra.Command, args []string) {
+			types := cdc.TypeInfosByName()
+			for k, _ := range types {
+				converter.Add(types[k].PtrToType)
```

Comment:
the typescriptify converter  wants a struct literal here but Im struggling to pull the type out of amino properly and getting nothing as a result